### ISSUE TITLE
Articles et bibliographie

### ISSUE THREAD

**Issue by ajaniak** [Chartes-TNAH/Boatlas#4]
@DUREAULT,

nous sommes d'accord que la capacité à lier des articles et une bibliographie revient à ajouter une catégorie "biblio" dans la description de chaque lieu? 

Merci pour ta réponse

**Comment by PonteIneptique** [comment]
@ajaniak @dureault : non, comme pour les lieux, il s'agit d'objets à part qui pourraient être liés à plusieurs lieux.

**Comment by ajaniak** [comment]
J'ai bien fait de poser la question alors...
Merci

**Comment by DUREAULT** [comment]
Oui effectivement, je l'avais compris comme ça aussi! 
Merci!
